configs/nginx/demo-app.conf.template

```diff
@@ -0,0 +1,29 @@
+server {
+    listen 443 ssl;
+    server_name demo.${CERTBOT_DOMAIN_SUFFIX};
+
+    ssl_certificate /etc/letsencrypt/live/${CERTBOT_DOMAIN_SUFFIX}/fullchain.pem;
+    ssl_certificate_key /etc/letsencrypt/live/${CERTBOT_DOMAIN_SUFFIX}/privkey.pem;
+
+    # Backend API
+    location /api/ {
```

Comment:
@parketh jfyi this mean the .env file needs to be changed locally

Reply:
wdym? as in the .env file for existing deployments?

Reply:
sry i mean you need to change the `packages/frontend/.env` on your local machine to have 

```
NEXT_PUBLIC_BACKEND_API_HOST=https://localhost:8000/api
```